averagesalary.py

```diff
@@ -0,0 +1,16 @@
+#https://leetcode.com/problems/average-salary-excluding-the-minimum-and-maximum-salary/
+#Given an array of unique integers salary where salary[i] is the salary of the employee i.
+#Return the average salary of employees excluding the minimum and maximum salary.
+class Solution:
+    def average(self, salary: List[int]) -> float:
```

Comment:
try to do this in fewer lines with these methods

https://www.tutorialspoint.com/python/list_max.htm
https://www.geeksforgeeks.org/find-average-list-python/